mobile-app/app/screens/TransactionAuthorization/PasscodePrompt.tsx

```diff
@@ -36,148 +40,140 @@ interface PasscodePromptProps {
 }
 
 const PromptContent = React.memo((props: PasscodePromptProps): JSX.Element => {
+  const { isLight } = useThemeContext()
   return (
     <>
-      <ThemedTouchableOpacity
-        dark={tailwind('bg-gray-900')}
-        light={tailwind('bg-white')}
+      <ThemedTouchableOpacityV2
+        light={tailwind('bg-mono-light-v2-100')}
+        dark={tailwind('bg-mono-dark-v2-100')}
         onPress={() => props.onCancel(USER_CANCELED)}
-        style={tailwind('items-end pt-2 pr-2')}
+        style={[tailwind('items-end pt-6 pr-6'), { borderTopLeftRadius: 15, borderTopRightRadius: 15 }]}
         testID='cancel_authorization'
         disabled={[TransactionStatus.BLOCK, TransactionStatus.SIGNING].includes(props.status)}
       >
-        <ThemedIcon
-          dark={tailwind('text-white')}
-          light={tailwind('text-black')}
-          iconType='MaterialIcons'
-          name='close'
-          size={26}
-        />
-      </ThemedTouchableOpacity>
-
-      <ThemedView
-        dark={tailwind('bg-gray-900')}
-        light={tailwind('bg-white')}
+        <View
+          style={[tailwind(`w-6 h-6 flex justify-center items-center rounded-full ${isLight ? 'border-mono-light-v2-900' : 'border-mono-dark-v2-900'}`), { borderWidth: 1.5 }]}
```

Comment:
use `ThemedViewV2` here

Reply:
We aren't using any theming there, though.

Reply:
here we used theme `isLight ? 'border-mono-light-v2-900' : 'border-mono-dark-v2-900'`

Reply:
Only for the borders not text but let's use anyways

Reply:
as we are using `<ThemedViewV2` we can add `light` and `dark` mode class as 
```
<ThemedViewV2
   light={tailwind('border-mono-light-v2-900')}
   dark={tailwind('border-mono-dark-v2-900')}
   style={[tailwind('w-6 h-6 flex justify-center items-center rounded-full' ), { borderWidth: 1.5  }]}
>
```